lib/requestability_resolver.js

```diff
@@ -0,0 +1,28 @@
+const DeliveryLocationsResolver = require('./delivery-locations-resolver')
+class RequestabilityResolver {
+  static fixItemRequestability (elasticSearchResponse) {
+    elasticSearchResponse.hits.hits
+      .map((hit) => {
+        hit._source.items = hit._source.items.map(function (item) {
+          if (item.electronicLocator) return item
+          if (item.recapCustomerCode && item.recapCustomerCode[0]) {
+            item = DeliveryLocationsResolver.attachRecapDeliveryInfo(item, item.recapCustomerCode[0])
```

Comment:
Given the name of this call, why pass in `item.recapCustomerCode` as second param when the resolver can get it from the first param?